### ISSUE TITLE
ionic v1.0 and cordova 5.0 compatibility during emulation

### ISSUE THREAD

**Issue by w-** [EricSimons/ionic-course#10]
This was near impossible to figure out and only after a few hours of near fruitless googling i stumbled upon the solution by accident.

**Symptoms:**
You attempt to emulate Android device using AVD or genymotion.
You get:
![app error](https://cloud.githubusercontent.com/assets/357317/7577973/68534450-f803-11e4-8110-8326dbd814d2.PNG)

On genymotion and actual device you get:
![blank](https://cloud.githubusercontent.com/assets/357317/7577911/d6a00b74-f802-11e4-9994-eecbbb1691f5.PNG)

and no matter what you do at signup or login the http post errors out giving you 
'Hmm... try another username.'
![alert](https://cloud.githubusercontent.com/assets/357317/7577906/ced61744-f802-11e4-8e5d-4fd14e43f7ce.png)

**Solution:**
http://forum.ionicframework.com/t/application-error-there-was-a-network-error/21999

Needs a whitelist plugin.
http://abou-kone.com/2015/04/25/ionic-there-was-a-network-error-when-running-on-genymotion/

Please include step of adding this plugin to tutorial accordingly.

Thanks

**Comment by PeteMugartegui** [comment]
Do you get this problem solved? I'm having a similar issue, I run it in my device (Moto G) but it doesn't matter what I do, I always get the message "Hmm... try another username", but when i run it with ionic serve everything works perfectly in the browser. 
Do you have any idea what it's causing this problem? it's breaking my head in two
Cheers

**Comment by PeteMugartegui** [comment]
For anyone who has this problem, here it is the solution: 
For me the current github version of whitelist plugin didn't work.
So I search for an older version of the plugin and you can install it by adding this line in the console in your project:

cordova plugin add cordova-plugin-whitelist@1.0.0

cheers.

**Comment by kencrocken** [comment]
At first I was not able to run the app with Genymotion, and got similar errors to this thread ... basically I was not able to sign in - all I got was the alert box, and the splash screen background was white.

I added the the cordova-plugin-whitelist and I was able to sign in to the app and skip/favorite songs, however the splash screen is still just white.  I tried adding the 1.0.0 version, and  its the same- the splash is still just white.  Any one else having this issue?

![image](https://cloud.githubusercontent.com/assets/7041191/9365953/7517d52a-4682-11e5-870f-69f97d72abd5.png)
